package.json

```diff
@@ -2,23 +2,32 @@
   "name": "browserify-rsa",
   "version": "4.0.1",
   "description": "RSA for browserify",
+  "keywords": [],
```

Comment:
we should really only add the keywords array if we're adding some keywords to the project